framework/resources/Functional/parquet_storage/dictionary/dictionary.json

```diff
@@ -0,0 +1,27 @@
+{
+  "testId": "parquet_dictionary_enc",
+  "type": "group",
+  "description": "Test querying parquet files in the dictionary encoding.",
+  "submit-type": "jdbc",
+  "categories": [
+    "functional"
+  ],
+  "matrices": [
+    {
+      "query-file": ".*.sql",
+      "schema": "dfs.drilltestdir",
+      "output-format": "tsv",
+      "expected-file": ".*.e_tsv",
+      "verification-type": [
+        "in-memory"
+      ]
+    }
+  ],
+  "datasources": [
+    {
+      "mode": "cp",
+      "src": "Datasources/parquet_storage/bigIntDictionary.parquet",
+      "dest": "/drill/testdata/bigIntDictionary.parquet"
```

Comment:
Can we please create a dir inside /drill/testdata? Like /drill/testdata/parquet/bigIntDictionary

Also, no need to provide /drill/testdata in the dest - any path specified is always relative to this location. For example, you can just provide parquet/bigIntDictionary

Reply:
Done.